### ISSUE TITLE
can i add this to winforms

### ISSUE THREAD

**Issue by HunterZREBORN** [icsharpcode/AvalonEdit#413]
hi

i was wondering how to add this to winforms,
and if there is a tutorial (im very bad at like wpf coding and xaml)

thx

**Comment by HunterZREBORN** [comment]
this is also my alt as im working on a website

**Comment by MrKillingHunter** [comment]
this is my main

**Comment by Philorrine** [comment]
yes you can , you can show WPF UI on winform by ElementHost, try it

> hi
> 
> i was wondering how to add this to winforms, and if there is a tutorial (im very bad at like wpf coding and xaml)
> 
> thx
